project/src/mocks/films.tsx

```diff
@@ -38,272 +18,327 @@ const duration: Duration = {
+    id: 'bohemian-rhapsody',
     name: 'Bohemian Rhapsody',
     genre: genre,
     year: year,
     director: director,
     actors: actors,
     duration: duration,
     description: description,
+    videoPath: videoPath,
     path: 'img/bohemian-rhapsody.jpg',
     posterPath: posterPath,
     backgroundPath: backgroundPath,
-    reviews: reviewsForFilm,
+    reviews: ReviewsForFilm,
     similarMovies: []
   },
   {
+    id: 'fantastic-beasts-the-crimes-of-grindelwald',
     name: 'Fantastic Beasts: The Crimes of Grindelwald',
     genre: genre,
     year: year,
     director: director,
     actors: actors,
     duration: duration,
     description: description,
+    videoPath: videoPath,
     path: 'img/fantastic-beasts-the-crimes-of-grindelwald.jpg',
     posterPath: posterPath,
     backgroundPath: backgroundPath,
-    reviews: reviewsForFilm,
+    reviews: ReviewsForFilm,
     similarMovies: []
   },
   {
+    id: 'macbeth',
     name: 'Macbeth',
     genre: genre,
     year: year,
     director: director,
     actors: actors,
     duration: duration,
     description: description,
+    videoPath: videoPath,
     path: 'img/macbeth.jpg',
     posterPath: posterPath,
     backgroundPath: backgroundPath,
-    reviews: reviewsForFilm,
+    reviews: ReviewsForFilm,
     similarMovies: []
   },
   {
+    id: 'aviator',
     name: 'Aviator',
     genre: genre,
     year: year,
     director: director,
     actors: actors,
     duration: duration,
     description: description,
+    videoPath: videoPath,
     path: 'img/aviator.jpg',
     posterPath: posterPath,
     backgroundPath: backgroundPath,
-    reviews: reviewsForFilm,
+    reviews: ReviewsForFilm,
     similarMovies: []
   },
   {
+    id: 'we-need-to-talk-about-kevin',
     name: 'We need to talk about Kevin',
     genre: genre,
     year: year,
     director: director,
     actors: actors,
     duration: duration,
     description: description,
+    videoPath: videoPath,
     path: 'img/we-need-to-talk-about-kevin.jpg',
     posterPath: posterPath,
     backgroundPath: backgroundPath,
-    reviews: reviewsForFilm,
+    reviews: ReviewsForFilm,
     similarMovies: []
   },
   {
+    id: 'what-we-do-in-the-shadows',
     name: 'What We Do in the Shadows',
     genre: genre,
     year: year,
     director: director,
     actors: actors,
     duration: duration,
     description: description,
+    videoPath: videoPath,
     path: 'img/what-we-do-in-the-shadows.jpg',
     posterPath: posterPath,
     backgroundPath: backgroundPath,
-    reviews: reviewsForFilm,
+    reviews: ReviewsForFilm,
     similarMovies: []
   },
   {
+    id: 'revenant',
     name: 'Revenant',
     genre: genre,
     year: year,
     director: director,
     actors: actors,
     duration: duration,
     description: description,
+    videoPath: videoPath,
     path: 'img/revenant.jpg',
     posterPath: posterPath,
     backgroundPath: backgroundPath,
-    reviews: reviewsForFilm,
+    reviews: ReviewsForFilm,
```

Comment:
Not the same reviews are for every single movie, just add a new object as a value and specify the comments here